examples/angular-example/.vscode/extensions.json

```
@@ -0,0 +1,4 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=827846
```

Comments aren't allowed this way in JSON. Is it possible to use either one of these two solutions instead?
- [`JSON5`](https://json5.org/)
- using a hack to define this like this:
```suggestion
  "_for_more_information_visit": "https://go.microsoft.com/fwlink/?linkid=827846"
```